src/Accounts/AccountGeneration.java

```diff
@@ -0,0 +1,17 @@
+package Accounts;
+
+public class AccountGeneration {
+    private static int id = 0;
```

Comment:
[Fix] poti adauga acest camp direct in clasa Account si sa il incrementezi in constructor, nu ai nevoie de o clasa suplimentara pentru generare de iduri. La fel si pentru celelalte clase generatoare de id